applications/FluidDynamicsApplication/custom_utilities/mass_conservation_utility.h

```diff
@@ -55,50 +53,44 @@ namespace Kratos
 /// Utility to modify497 the distances of an embedded object in order to avoid bad intersections
 /// Besides, it also deactivate the full negative distance elements
```

Comment:
I think we can take the chance to correct and improve this.

Reply:
improved